ocs_ci/ocs/utils.py

```diff
@@ -887,7 +887,12 @@ def run_must_gather(log_dir_path, image, command=None):
     # https://github.com/red-hat-storage/ocs-ci/issues/3240
     mg_output = ""
     ocs_version = version.get_semantic_ocs_version_from_config()
-    timeout = 1500 if ocs_version >= version.VERSION_4_6 else 600
+    if version.VERSION_4_6 < ocs_version < version.VERSION_4_11:
+        timeout = 1500
+    elif ocs_version >= version.VERSION_4_11:
+        timeout = 2700
+    else:
+        log.error("OCS/ODF version isn't supported")
```

Comment:
if version is not supported it may be better to raise an exception instead of log error